walk-in-interview/src/main/webapp/strings.en.js

```diff
@@ -62,20 +62,20 @@ AppStrings['log-in'] = {
 };
 
 /**
- * All the strings to be used in the log-in/business-log-in.html.
+ * All the strings to be used in the log-in/business/index.html.
  */
 AppStrings['business-log-in'] = {
   'account': 'Account (Email)',
   'password': 'Password',
 };
 
 /**
- * All the strings to be used in the log-in/applicant-log-in.html.
+ * All the strings to be used in the log-in/applicant/index.html.
  */
 AppStrings['applicant-log-in'] = {
   'account': 'Account (Phone Number)',
-  'password': 'Password',
-  'otp': 'Get OTP instead',
+  'password': 'OTP',
```

Comment:
nit: maybe change this to 'opt' and then the below one to 'otp-button'?

Reply:
Yes OTP and password are two very different concepts one is long term secret, one is short term token. Mixing this is very error prone.